py-data.csv

```diff
@@ -136,9 +136,9 @@ https://github.com/chaosmail/python-fs,2567922ced9387e327e65f3244caff3b7af35684,
 https://github.com/chaosmail/python-fs,2567922ced9387e327e65f3244caff3b7af35684,fs/tests/test_touch.py::test_touch_on_new_file,NIO,Accepted,https://github.com/chaosmail/python-fs/pull/9,
 https://github.com/chie8842/expstock,28b7ece83fa601efd2b6c4b8b7bdd1a3b9ebc1df,expstock/test_expstock.py::test_append_param,NIO,Opened,https://github.com/chie8842/expstock/pull/18,
 https://github.com/chinapandaman/PyPDFForm,fcc1b297005a39900c1f928f602cbc5f38a1d60e,tests/unit/test_font.py::test_register_font_and_is_registered,NIO,Accepted,https://github.com/chinapandaman/PyPDFForm/pull/254,
-https://github.com/chinapnr/fishbase,1615ccff9106bf42fc6ae862cd13353c12c1b76d,test/test_logger.py::TestFishLogging::test_format3,OD-Brit,,,
+https://github.com/chinapnr/fishbase,1615ccff9106bf42fc6ae862cd13353c12c1b76d,test/test_logger.py::TestFishLogging::test_format3,OD-Brit,MovedOrRenamed,,Test renamed to test_format_03
```

Comment:
Please add new rows for the renamed tests

Reply:
Added a new row for the renamed test- I noticed there was a second duplicate instance of the same test. I've removed that row. Please let me know if that's alright

Reply:
That's alright, seems like the two tests are duplicated (Although the repo URL isn't the same, they'll direct to the same repo)